modules/ztp-pre-staging-downloading-artifacts.adoc

```diff
@@ -0,0 +1,416 @@
+    - catalog: eko4.cloud.lab.eng.bos.redhat.com:8443/redhat/certified-operator-index:v4.11 <1>
+----
+<1> The catalog source is changed from an official Red Hat registry.
+// On line 315 and 341 we have the same field with the same value, equal identation. Is that intentional? If yes, what does each one do?
+
+When you download images by using a local or disconnected registry, you have to first add certificates for the registries that you want to pull content from.
+
+. To avoid any errors, copy the registry certificate into your server:
++
+[source,terminal]
+----
+# cp /tmp/eko4-ca.crt /etc/pki/ca-trust/source/anchors/.
+----
+
+. Then, update the certificates trust store:
++
+[source,terminal]
+----
+# update-ca-trust
+----
+
+. Mount the host `/etc/pki` folder into the factory-cli image:
++
+[source,terminal]
+----
+# podman run -v /mnt:/mnt -v /root/.docker:/root/.docker -v /etc/pki:/etc/pki --privileged -it --rm quay.io/user/telco-ran-tools:acm-versioning -- \
+    factory-precaching-cli download -r 4.11.5 -f /mnt \
+    --ai-img registry.redhat.io/multicluster-engine/assisted-installer-agent-rhel8@sha256:da1753f9fcb9e229d0a68de03fac90d15023e647a8db531ae489eb93845d5306 \
+    --ai-img registry.redhat.io/multicluster-engine/assisted-installer-reporter-rhel8@sha256:e8d6b78248352b1a8e05a22308185a468d4a139682d997a7f968b329abbc02cd \
+    --ai-img registry.redhat.io/multicluster-engine/assisted-installer-rhel8@sha256:33abd6e21cfdc36dd4337fa6f3c3442d33fc3f976471614dca5b8ef749e7a027 \
+    --img quay.io/user/troubleshoot \
+    --du-profile --acm-hub 2.5.4 -s \
+    --skip-imageset
+----
+
+////
+Could we have an example where this goes through? I feel like it makes more sense to include adding the certs before the user encounters the error. Let's point it out and add the steps here. I can include a troubleshooting section in case they skip adding the certs
+
+Also, if we do it this way, the remaining step might not be required, if I understand it correctly.
+////
+
+. Download the images without generating a new `imageSetConfiguration` CR:
++
+[source,terminal]
+----
+# podman run -v /mnt:/mnt -v /root/.docker:/root/.docker --privileged -it --rm quay.io/user/telco-ran-tools:acm-versioning -- factory-precaching-cli \
+    download -r 4.11.5 -f /mnt \
+    --ai-img registry.redhat.io/multicluster-engine/assisted-installer-agent-rhel8@sha256:da1753f9fcb9e229d0a68de03fac90d15023e647a8db531ae489eb93845d5306 \
+    --ai-img registry.redhat.io/multicluster-engine/assisted-installer-reporter-rhel8@sha256:e8d6b78248352b1a8e05a22308185a468d4a139682d997a7f968b329abbc02cd \
+    --ai-img registry.redhat.io/multicluster-engine/assisted-installer-rhel8@sha256:33abd6e21cfdc36dd4337fa6f3c3442d33fc3f976471614dca5b8ef749e7a027 \
+    --img quay.io/user/troubleshoot \
+    --du-profile --acm-hub 2.5.4 -s \
+    --skip-imageset <1>
+----
+<1> The `--skip-imageset` prevents a new `imageSetConfiguration` CR from being generated.
+
+.Example output
++
```

Comment:
This example from the download.md doc is part of a "warning" section to show the error that occurs if you're trying to do the downloads from a disconnected registry without having the proper certificate installed.